### PR TITLE
Added more detailed description and some missing details.

### PR DESCRIPTION
**Pull request by nizamial09**
Followed the design pattern shown in https://projects.invisionapp.com/share/ABTPZXYKYE9#/screens/382233629_1-1-OCS-Operator-Install

Fixes : 
1. https://bugzilla.redhat.com/show_bug.cgi?id=1760386
2. https://bugzilla.redhat.com/show_bug.cgi?id=1784093

Changed the description of the OCS install page in the OperatorHub by adding some more detailed
descriptions.

Added some of the missing details like REPOSITORY, CONTINER IMAGE, CREATED AT and SUPPORT details on the install page.

To support the Capability Level in the UI, https://github.com/openshift/console/pull/3896 needs to be merged into console.

Signed-off-by: Nizamudeen <nia@redhat.com>

### CODE REVIEW

**Comment by nizamial09** [comment]
@umangachapagain Please review this one. Thanks

File: deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -56,6 +56,12 @@ metadata:
       ]
     capabilities: Full Lifecycle
     categories: Storage
+    containerImage: quay.io/rhceph-dev/ocs-operator:4.3.0
```
Comment (jarrpa) [inline deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml]: This is not correct. We should not be pointing at the downstream images from here. Only use images in `quay.io/ocs-dev`.

File: deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -286,15 +292,11 @@ spec:
...
-
-    ## Components
-
-    OpenShift Container Storage deploys three operators.
+    **Red Hat OpenShift Container Storage** deploys three operators.
 
     ### OpenShift Container Storage operator
 
-    The OpenShift Container Storage operator is the primary operator for OpenShift Container Storage. It serves to facilitate the other operators in OpenShift Container Storage by performing administrative tasks outside their scope as well as watching and configuring their CustomResources.
+    OpenShift Container Storage operator is the primary operator for OpenShift Container Storage. It serves to facilitate the other operators in OpenShift Container Storage by performing administrative tasks outside their scope as well as watching and configuring their CustomResources.
```
Comment (jarrpa) [inline deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml]: "The" should not be removed here, this is grammatically incorrect.

File: deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -304,9 +306,25 @@ spec:
...
+    * **Elastic storage in your datacenter:** Scale is now possible in your datacenter. Get started with a few terabytes, and easily scale up. Simply add more nodes and Rook takes care of the rest.
+
+    * **Simplified data management:** Easily create hybrid and multi-cloud data storage for your workloads, using a single namespace.
+
+    # Pre-requisites
+
     ## Before Subscription
 
-    Before subscribing to OpenShift Container Storage, there are two pre-requisites that need to be satisfied.
+    Before subscribing to OpenShift Container Storage, the following pre-requisites are to be satisfied.
```
Comment (jarrpa) [inline deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml]: s/are to/must/

File: deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml [hunk 4]
```diff
@@ -304,9 +306,25 @@ spec:
 
     The NooBaa operator deploys and manages the [NooBaa][2] Multi-Cloud Gateway on OpenShift, which provides object storage.
 
+    # Core Capabilities
+
+    * **Self-managing service:** Whichever supported storage technologies you choose, OpenShift Container Storage ensures that resources can be deployed and managed automatically.
```
Comment (jarrpa) [inline deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml]: s/Whichever/No matter which/

File: deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml [hunk 5]
```diff
@@ -304,9 +306,25 @@ spec:
...
+
+    * **Hyper-scale or hyper-converged:** With OpenShift Container Storage you can either build dedicated storage clusters or hyper-converged clusters where your apps run alongside storage.
+
+    * **File, Block, and Object provided by OpenShift Container Storage:** OpenShift Container Storage integrates Ceph with multiple storage presentations including object storage (compatible with S3), block storage, and POSIX-compliant shared file system.
+
+    * **Your data, protected:** OpenShift Container Storage efficiently distributes and replicates your data across your cluster to minimize the risk of data loss. With snapshots, cloning, and versioning, no more losing sleep over your data.
+
+    * **Elastic storage in your datacenter:** Scale is now possible in your datacenter. Get started with a few terabytes, and easily scale up. Simply add more nodes and Rook takes care of the rest.
```
Comment (jarrpa) [inline deploy/olm-catalog/ocs-operator/4.3.0/ocs-operator.v4.3.0.clusterserviceversion.yaml]: `Simply add more nodes and Rook takes care of the rest.` This is not true. Remove this sentence.

**Pull request merged**